src/FileSystem/PhpFilesFinder.php

```diff
@@ -25,7 +25,7 @@ public function findInDirectoriesAndFiles(array $source): array
             }
         }
 
-        if (count($directories)) {
+        if (count($directories) > 0) {
```

Comment:
This suggested there might be minus count of directories and that it should be tested.

:-1: